src/tools/wat2wasm.cc

```diff
@@ -80,6 +85,12 @@ static void ParseOptions(int argc, char* argv[]) {
   parser.AddOption('o', "output", "FILE",
                    "Output wasm binary file. Use \"-\" to write to stdout.",
                    [](const char* argument) { s_outfile = argument; });
+  parser.AddOption('p', "inputfile", "PATHFILE",
+                   "Input binary file. Use \"-\" to get data.",
```

Comment:
How about "Input binary for custom section" for the docs and "custom_section_filename" for the name?